cmd/pullrequest.go

```diff
@@ -43,4 +47,24 @@ var pullrequestCmd = &cobra.Command{
 
 func init() {
 	rootCmd.AddCommand(pullrequestCmd)
+	
+	pullrequestCmd.Flags().StringVarP(
+		&LinkType,
+		"link-type",
+		"l",
+		"close",
+		"The type of link to create with the associated issue.",
+	)
+}
+
+func validateLinkType() {
+	validLinkTypes := map[string]bool {
+		"close": true,
+		"connect": true,
+	}
+
+	if !validLinkTypes[LinkType] {
```

Comment:
Why isn't `LinkType` passed into the `validateLinkType` rather than being used as a global-ish const?

Reply:
Addressed by moving validation into util class.